x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/IndexLifecycleExplainResponseTests.java

```diff
@@ -94,6 +101,21 @@ public void testInvalidStepDetails() {
         assertThat(exception.getMessage(), containsString("=null"));
     }
 
+    public void testTimeSinceIndexCreation() {
```

Comment:
shall we also test `getAge` is calculated using `lifecycleDate` ?

Reply:
Expanded the test case to test both dates